<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_equipment</class>
 <widget class="QMainWindow" name="add_equipment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Επεξεργασία Εργαστηριακού Εξοπλισμού</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>media/gear.png</normaloff>media/gear.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Calligraphy</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Συμπληρώστε τα στοίχεια για επεξεργασία εξοπλισμού.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ακριβής Ονομασία:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Μοντέλο:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Βασικά Χαρακτηριστικά :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Χρόνος Κτήσης:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Σειριακός Αριθμός:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Σκοπός Χρήσης:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ποσότητα:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Κατάσταση:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Βάρος:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Διαθεσιμότητα:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Λέξεις Κλεδιά(Χωρίστε με κόμμα):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Κατηγορία:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Υποκατηγορία:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_equipment_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>350</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Δυνατότητα Μετακίνησης:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_equipment_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>535</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Επεξεργασία Εξοπλισμού</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_equipment_fields">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>535</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Καθαρισμός </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_12">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_13">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equipment_14">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
